### PR TITLE
Adds a damage threshold for robolimbs to count as malfunctioning

### PR DESCRIPTION
**Pull request by aceedex**
[qol] [balance]

what it says on the tin. a robotic limb must have a total combined damage of 15 or higher for the limb to begin dropping things. This is to prevent situations where youre exposed for a picosecond to vaccum and suddenly youre hurting for a welder and wires, or when an assistant accidentally punches you, or a vending machine nails you with the dans soda

:cl:
 * rscadd: robotic limbs now have a 15 damage threshold, burn and brute combined, before they are considered malfunctioning

### CODE REVIEW

**Comment by aceedex** [comment]
I wanted to keep it separate in case there are other things that use is_malfunctioning, but if thats what it needs that what it needs

**Comment by Hoshi-chan** [comment]
Finally, thank fuck

**Comment by aceedex** [comment]
well, thank you robo. there was no way in hell I'd have figured that out in this lifetime

File: code/modules/organs/organ_external.dm [hunk 1]
```diff
@@ -1127,7 +1127,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 
 //Is the limb robotic and malfunctioning
 /datum/organ/external/proc/is_malfunctioning()
-	return (is_robotic() && ((status & ORGAN_MALFUNCTIONING) || prob(brute_dam + burn_dam)))
+	return (return (is_robotic() && ((status & ORGAN_MALFUNCTIONING) || (brute_dam + burn dam > 14 && prob(brute_dam + burn_dam)))))
```
Comment (MadmanMartian) [inline code/modules/organs/organ_external.dm]: change it to
`	return (is_robotic() && (status & (ORGAN_MALFUNCTIONING) || (brute_dam + burn_dam > min_broken_damage/2 && prob(brute_dam + burn_dam))))`

That'll work, and unsnowflake that 14 to something we can work with

**Comment by Hoshi-chan** [comment]
When is this getting merged? It's been 5 days.

**Comment by MadmanMartian** [comment]
Guess I'll do it.

**Pull request merged**